ansible/templates/common/virtualenv.sh.j2

```diff
@@ -1,14 +1,18 @@
 {% if doc %}
 Create the virtual environment. This is the preferred way to install OMERO.web::
+
+# option 1: in a virtual environment with --system-site-packages on
+virtualenv {{ virtualenv_path }} --system-site-packages
+
+# option 2: in a virtual environment with --system-site-packages off
+virtualenv {{ virtualenv_path }} --no-site-packages
```

Comment:
Do you need the "--no-site-packages" flag? Isn't this the default?

Reply:
removing this I wanted to be more explicit but it is deprecated in newer version